UI/GameSettingsScreen.cpp

```diff
@@ -1072,6 +1082,25 @@ UI::EventReturn GameSettingsScreen::OnChangeMacAddress(UI::EventParams &e) {
 	return UI::EVENT_DONE;
 }
 
+UI::EventReturn GameSettingsScreen::OnChangeRemoteISOSubdir(UI::EventParams &e) {
+	//Conform to HTTP standards
+	for (size_t i = 0; i < g_Config.sRemoteISOSubdir.length(); i++)
+	{
+		if (g_Config.sRemoteISOSubdir[i] == ' ') g_Config.sRemoteISOSubdir[i] = '+';
+		if (g_Config.sRemoteISOSubdir[i] == '\\') g_Config.sRemoteISOSubdir[i] = '/';
+		if (g_Config.sRemoteISOSubdir[i] == '?') {
+			g_Config.sRemoteISOSubdir.erase(i); //truncate for safety
+			break;
+		}
+	}
+	//Make sure it begins and ends with /
+	if (g_Config.sRemoteISOSubdir[0] != '/')
+		g_Config.sRemoteISOSubdir = "/" + g_Config.sRemoteISOSubdir;
+	if (g_Config.sRemoteISOSubdir[g_Config.sRemoteISOSubdir.length() - 1] != '/')
```

Comment:
This one is a little more difficult.  If the user gives me http://server:80/xyz is xyz a directory or cgi script.  If it is a directory then I have to put a / before the filename, if it is a cgi then I have to strip it off.  I'll have to think about that.